game_board.py

```diff
@@ -0,0 +1,99 @@
+import pyglet
+
+
+class Cell:
+    def __init__(self, position_x, position_y, cell_size, state):
+        self.position_x = position_x    # position is bottom left corner
+        self.position_y = position_y
+        self.cell_size = cell_size
+        self.state = state
+
+    def add_cell_to_batch(self, batch):
+        left = self.position_x
+        right = self.position_x + self.cell_size
+        bottom = self.position_y
+        top = self.position_y + self.cell_size
+
+        vertex_list = batch.add(5, pyglet.gl.GL_LINE_STRIP, None,
+                                ('v2i', (left, bottom,
+                                         right, bottom,
+                                         right, top,
+                                         left, top,
+                                         left, bottom)),
+                                ('c3B', (0, 0, 255,
+                                         0, 0, 255,
+                                         0, 0, 255,
+                                         0, 0, 255,
+                                         0, 0, 255)))
```

Comment:
The following comment is related to all of your code. The way you write is PEP-8 compliant; it is unfortunately also the worst from the cognitive and practical sense.

Cognitively - I can't find the article now, but basically "the further in the code line, the less understandable the information is" - even including just indented lines.

Practically: we already are using the prohibitive 80 (79, really) char limit; if you break the line like here, you have some 40 chars remaining. If you add a single indentation level, you have 68 chars remaining.

I strongly suggest using the following indentation style:

```
vertex_list = batch.add(
    5, 
    pyglet.gl.GL_LINE_STRIP, 
    None,
    ('v2i', (
        left, bottom, right, bottom, right, top, left, top, left, bottom)),
    ('c3B', (0, 0, 255, 0, 0, 255, 0, 0, 255, 0, 0, 255, 0, 0, 255))
)
```

Reply:
Changed